jaxb-ri/codemodel/codemodel/src/main/java/com/sun/codemodel/JFormatter.java

```diff
@@ -75,21 +84,33 @@
      *
      * @param space
      *        Incremental indentation string, similar to tab value.
+     *
+     * @param classNameReplacer
+     *        Class names to replace.
      */
-    public JFormatter(PrintWriter s, String space) {
+    public JFormatter(PrintWriter s, String space, Map<String, String> classNameReplacer) {
```

Comment:
this is public API, so instead of altering existing constructor, new one should be created

Reply:
Good point, I have duplicated it.